services/QuillLMS/app/controllers/ortto_integration/webhooks_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module OrttoIntegration
+  class WebhooksController < ApplicationController
+    protect_from_forgery except: :create
+    before_action :valid_params?, only: [:create]
+
+    class OrttoWebhookBadRequestException < StandardError; end
+
+    VALID_ACTIONS = [
+      SUBSCRIBE   = 'subscribe',
+      UNSUBSCRIBE = 'unsubscribe'
+    ]
+
+    def create
```

Comment:
With webhooks, there's usually some security signature verification (e.g. [stripe](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/app/controllers/stripe_integration/webhooks_controller.rb#L20) or [intercom](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/app/controllers/intercom_integration/webhooks_controller.rb#L70).  There probably should be some kind of security check for incoming requests.

Reply:
Forgot to mention - there's been some issues configuring the webhook within Ortto's UI. The webhook won't be used immediately upon launch, so Peter Sharkey is fine with configuring the webhook after the rest of the code is deployed.